### DISCUSSION TITLE
Migrate from marctv docker container hosted on Synology NAS including plugins ?

### DISCUSSION THREAD

**Discussion by siw1973** [itzg/docker-minecraft-server#2112] [Q&A] [answered]
I'm trying to migrate two fully functioning marctv docker containers, with more coming, on a Synology NAS to itzg containers as I'd like to make use of the itzg Bungeecord Docker container to centralise a bit of the control, triggered by installing Plan Analytics yesterday.

I've managed to migrate the worlds across, but they do not load any of the Plugins.

Is there a way of doing this ? Or am I having to install them from scratch ?

**Comment by itzg** [comment]
There's quite a few ways to bring plugins into the container: https://github.com/itzg/docker-minecraft-server#working-with-mods-and-plugins. What have you tried?

**Reply by siw1973** [reply, marked as answer]
Thanks for the pointer..... TBH I searched the discussions using migrate and came up with the Docker Compose Simple, which didn't work and the tried creating a base container and pointing to the old data source.

I FULLY read the Readme and realised I'd made the school boy error of not specifying that the itzg containers needed to be specifically created as Parpermc....... As soon as I did that......

Sorry for wasting your time, all done now, onwards and upwards !
